cmd/celestia/bridge/bridge.go

```diff
@@ -0,0 +1,54 @@
+package bridge
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	cmdnode "github.com/celestiaorg/celestia-node/cmd"
+	"github.com/celestiaorg/celestia-node/nodebuilder/core"
+	"github.com/celestiaorg/celestia-node/nodebuilder/gateway"
+	"github.com/celestiaorg/celestia-node/nodebuilder/node"
+	"github.com/celestiaorg/celestia-node/nodebuilder/p2p"
+	"github.com/celestiaorg/celestia-node/nodebuilder/rpc"
+	"github.com/celestiaorg/celestia-node/nodebuilder/state"
+)
+
+// NOTE: We should always ensure that the added Flags below are parsed somewhere, like in the
+// PersistentPreRun func on parent command.
+
+func WithSubcommands() func(*cobra.Command, []*pflag.FlagSet) {
+	return func(c *cobra.Command, flags []*pflag.FlagSet) {
+		c.AddCommand(
+			cmdnode.Init(flags...),
+			cmdnode.Start(cmdnode.WithFlagSet(flags)),
+			cmdnode.AuthCmd(flags...),
+			cmdnode.ResetStore(flags...),
+			cmdnode.RemoveConfigCmd(flags...),
+			cmdnode.UpdateConfigCmd(flags...),
+		)
+	}
+}
+
+func NewCommand(options ...func(*cobra.Command, []*pflag.FlagSet)) *cobra.Command {
+	flags := []*pflag.FlagSet{
+		cmdnode.NodeFlags(),
+		p2p.Flags(),
+		core.Flags(),
+		cmdnode.MiscFlags(),
+		rpc.Flags(),
+		gateway.Flags(),
+		state.Flags(),
```

Comment:
my one nit, can we ensure these are ordered in the same order for each of the three new packages (so its easy to side by side and see which is different/missing)? I think they are almost there, jsut cmdnode and core flipped in one place

Reply:
Done 👍🏽